src/test-jdk17/java/com/fasterxml/jackson/databind/records/RecordWithJsonIgnoreTest.java

```diff
@@ -63,12 +64,6 @@ public void testSerializeJsonIgnoreAndJsonPropertyRecord() throws Exception {
         assertEquals("{\"id\":123}", json);
     }
 
-    @Test
-    public void testDeserializeJsonIgnoreAndJsonPropertyRecord() throws Exception {
-        RecordWithIgnoreJsonProperty value = MAPPER.readValue("{\"id\":123,\"name\":\"Bob\"}", RecordWithIgnoreJsonProperty.class);
-        assertEquals(new RecordWithIgnoreJsonProperty(123, "Bob"), value);
```

Comment:
Ok so this test was wrong, incorrectly asserting wrong expectation.

Reply:
Exactly